website/docs/usage/rule-based-matching.md

```diff
@@ -63,7 +63,7 @@ another token that's at least 10 characters long.
 
 spaCy features a rule-matching engine, the [`Matcher`](/api/matcher), that
 operates over tokens, similar to regular expressions. The rules can refer to
-token annotations (e.g. the token `text` or `tag_`, and flags (e.g. `IS_PUNCT`).
+token annotations (e.g. the token `text` or `tag_`, and flags (e.g. `IS_PUNCT`)).
```

Comment:
This is probably even better to avoid the double parentheses:

```suggestion
token annotations (e.g. the token `text` or `tag_`, and flags like `IS_PUNCT`).
```